cli/module_graph.rs

```diff
@@ -24,6 +25,15 @@ use std::hash::BuildHasher;
 use std::path::PathBuf;
 use std::pin::Pin;
 
+fn err_with_location(e: ErrBox, location: Location) -> ErrBox {
+  let location_str = format!(
+    "\nImported from \"{}:{}\"",
+    location.filename, location.line
+  );
+  let err_str = e.to_string();
+  OpError::other(format!("{}{}", err_str, location_str)).into()
+}
```

Comment:
Good! Consider adding a docstring for this method and a maybe TODO to have a more structured way of wrapping errors?